src/fmu/tools/qcforward/qcforward.py

```diff
@@ -1,12 +1,119 @@
+"""The qcforward module"""
+from __future__ import absolute_import, division, print_function  # PY2
+
+import sys
+from os.path import join
+
+import fmu.tools
+import yaml
+from . import _wellzonation_vs_grid as _wzong
+from . import _grid_statistics as _gstat
+
+
 class QCForward(object):
+    """
+    The QCforward class which has a set of QC functions that can be ran from
+    either RMS python, or on disk. The input `data` will be
+    somewhat different for the two run environents.
+
+    It should be easy to add new functions to this class. The idea is to reuse
+    as much as possible, and principles are:
+
+    * For the client (user), the callings scripts shall be lean
+
+    * All methods shal have a rich documention with examples, i.e. it shall
+      be possible for users with less skills in scripting to copy/paste and then modify
+      to their needs.
+
+    """
+
     def __init__(self):
-        self._name = None
-        self._whatever = None
+        self._method = None
+        self._data = None
+
+        self._verbosity = 0
+        self._project = None  # Roxar API project
+        self._path = "."
+
+        self._grid = None  # the primary XTGeo Grid() object
```

Comment:
These variables seems very specific while the QCForward class is quite generic.
E.g. many qc functions will not require a grid.
Perhaps these variables should be set elsewhere?

Reply:
I will consider. Otherwise a self._grid = None will tok take any space